lib/ansible/module_utils/common/text/formatters.py

```diff
@@ -38,7 +38,20 @@ def lenient_lowercase(lst):
 
 def human_to_bytes(number, default_unit=None, isbits=False):
     """Convert number in string format into bytes (ex: '2K' => 2048) or using unit argument.
```

Comment:
Please add separation from the docstring title line.
```suggestion
    """Convert number in string format into bytes (ex: '2K' => 2048) or using unit argument.

```